go.mod

```diff
@@ -102,6 +103,7 @@ require (
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.7.0 // indirect
 	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/josephspurrier/goversioninfo v1.4.0 // indirect
```

Comment:
Looks like the use of a `tools.go` file now causes all the tools used in go-winio to be imported; I see `tools/mkwinsyscall`, `github.com/akavel/rsrc `, as well as the other tools used in that repo 😞 

I tried to look if we could get rid of those (see https://github.com/microsoft/go-winio/pull/266), but it looks like there's even more things than I thought were added as dependency because of that.

Perhaps that `tools` should be a separate module.

Reply:
Hi, you means to reduce the "github.com/josephspurrier/goversioninfo" import?
Would you have any ideas about that.
It is imported by 'go mod tidy':sweat_smile::sweat_smile: